sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -176,7 +180,13 @@ function parseAtomResult(
     return result;
   }
 
-  throw new Error("Unrecognized Atom XML result: " + JSON.stringify(atomResponseInJson));
+  throw new RestError(
+    "Unrecognized Atom XML result: " + JSON.stringify(atomResponseInJson),
```

Comment:
Can we have this error message and the others that are thrown when parsing service output use similar verbiage?
The first sentence can be common and the second can provide more details.

Using the text from the serviceBusAtomManagementClient.ts error message, this can be

> Response from service is in bad format. Expected response to be in Atom XML format and have either feed or entry components.

?

cc @bterlson

Reply:
Updated